lib-find-meal/src/main/java/com/kinandcarta/lib/find/meal/adapter/MealsAdapter.kt

```diff
@@ -52,7 +53,7 @@ class MealsAdapter(var mealsList: List<Meal>) :
             }
 
             // TODO calculate distance (this could be done in the shared layer)
-            distanceView.text = "8637.35km"
+            distanceView.text = "${meal.distance} ${distanceUnit}"
 
             // TODO format these dates
```

Comment:
I'm currently working on formatting dates in the shared area also so can remove this :-)